### PR TITLE
Redirect to the return URL when go to the Login page, but it is already logged

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Redirect to the return URL when go to the Login page, but it is already logged

#### How should this be manually tested?
[Click here to access the workspace.](https://waza--storecomponents.myvtex.com/)

#### Types of changes
- [X] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/LoginContent.js [hunk 1]
```diff
@@ -193,21 +197,25 @@ class LoginContent extends Component {
     })
   }
 
+  redirect = () => {
+    this.props.runtime.navigate({
+      page: `store${this.state.returnUrl}`,
```
Comment (claudivanfilho) [inline react/LoginContent.js]: use `to`
```
...navigate({
  to: this.state.returnUrl,
})
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Get user profile when retrieve just the user's e-mail.
```
Comment (luciannojunior) [inline CHANGELOG.md]: This message is not clear. Perhaps "Now retrieving the user's profile instead of just its email"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Redirect to the return URL when go to the Login page, but it's already logged.
```
Comment (luciannojunior) [inline CHANGELOG.md]: Same. Maybe "Redirecting to the previous route in the app when the user tries to go to Login when already logged"?

File: react/utils/profile.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { path } from 'ramda'
+
```
Comment (luciannojunior) [inline react/utils/profile.js]: JSDoc explaining why is such logic necessary...

File: react/LoginContent.js [hunk 5]
```diff
@@ -250,9 +258,15 @@ class LoginContent extends Component {
       isInitialScreenOptionOnly,
       defaultOption,
       data: { loading },
+      session,
     } = this.props
     const { isOnInitialScreen } = this.state
 
+    if (getProfile(session) && !profile) {
```
Comment (luciannojunior) [inline react/LoginContent.js]: It is not clear, reading all the function calling, why is this necessary. May add an line of comment above this if statement? I mean "if that is profile (getProfile) but there is not profile (profile) do that"...

File: react/utils/profile.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+export const getProfile = data => {
+  let profile = path(['getSession', 'profile'], data)
+
+  if (!path(['id'], profile) && !path(['email'], profile)) {
+    profile = null
```
Comment (gugabribeiro) [inline react/utils/profile.js]: It is not better to return an `undefined` value instead of a `null` one? With this solution the function will be reduced to a return statement of the fourth line only.
